### ISSUE TITLE
How to exit REPL gracefully without killing the terminal

### ISSUE THREAD

**Issue by garyhlai** [BetterThanTomorrow/calva#1387]
How does one shut down REPL gracefully? I cannot seem to shut it down without killing the terminal entirely. If I try to exit via `ctrl + c` in the terminal, it'd first show me "Jack-in process exited. Status: null" but it just hangs there without giving back my terminal. If I press `ctrl + c` again, it'd just keep showing me "Killing the Jack-in process" each time I press (see screenshot).

Steps to reproduce:

1. create a project with `lein create app`
2. vscode command palette --> `start a project REPL and connect (aka jack-in)`
3. try to exit via `ctrl + c` in the terminal (zsh shell)

![Screen Shot 2021-11-17 at 5 51 39 PM](https://user-images.githubusercontent.com/22721482/142295413-175493e9-f758-4766-8fd1-6e237a5d5d1f.png)

Thank you for your help!

**Comment by PEZ** [comment]
Hello! Is the REPL still running after you have killed it the first time?

I suspect this might be about bad Ux. We should say something like **You can close this terminal now**. And something else at the following `ctrl+c` events. Like **Jack in process already killed. You can close this terminal.**

**Comment by garyhlai** [comment]
> Hello! Is the REPL still running after you have killed it the first time?
> 
> I suspect this might be about bad Ux. We should say something like **You can close this terminal now**. And something else at the following `ctrl+c` events. Like **Jack in process already killed. You can close this terminal.**

Hi! REPL is disconnected successfully when I killed it the first time. It doesn't say **"You can close this terminal now"** -- as can be seen from the screenshot above, it instead says **"Jack-in process exited. Status: null"** without giving back my terminal. It just hangs there. So if I try to `ctrl+c` again, it'd just keep saying **"Killing the Jack-in process".**

**Comment by PEZ** [comment]
Yes. Let me try again:

Calva could do this better.

> without giving back my terminal

This is purely a jack-in terminal. There is nothing to give back to you. When you have killed the process, you can close that terminal if you are not interested in the output there.

**Comment by bpringe** [comment]
I created an issue for improving the output in the jack-in terminal.
